### PR TITLE
Added a setting for MultiBoilerplate

### PR DESCRIPTION
**Pull request by Njsmooth**
I added in a configuration option for MultiBoilerplate that wasn't already there (maybe add it to managewiki?). Also, there seemed to be duplicate configurations as well as some formatting oddities, so I changed them to be a bit more uniform.

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -3385,10 +3380,14 @@
 
 	// MultiBoilerplate
 	'wgMultiBoilerplateDisplaySpecialPage' => [
-		'wmgUseMultiBoilerplate' => true,
```
Comment (Universal-Omega) [inline LocalSettings.php:3388]: This should still be wmgUseMultiBoilerplate

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (Njsmooth) [reply LocalSettings.php:3388]: Did the change I make do it, or do I need to change all of them?

File: LocalSettings.php [hunk 2]
```diff
@@ -3388,7 +3383,11 @@
 		'wmgUseMultiBoilerplate' => true,
 	],
 	'wgMultiBoilerplateOptions' => [
-		'wmgUseMultiBoilerplate' => false,
```
Comment (Universal-Omega) [inline LocalSettings.php:3391]: This should still be wmgUseMultiBoilerplate also.

File: LocalSettings.php [hunk 3]
```diff
@@ -3388,7 +3383,11 @@
...
+		'default' => false,
+	],
+	'wgMultiBoilerplateOverwrite' => [
+		'default' => false,
```
Comment (Universal-Omega) [inline LocalSettings.php:3389]: wmgUseMultiBoilerplate

File: LocalSettings.php [hunk 4]
```diff
@@ -3388,7 +3383,11 @@
 		'wmgUseMultiBoilerplate' => true,
```
Comment (Universal-Omega) [inline LocalSettings.php:3383]: Should default to false.

**Review by Universal-Omega — DISMISSED** [review]

**Pull request merged**